bokehjs/src/lib/embed/index.ts

```diff
@@ -15,6 +15,21 @@ export {add_document_from_session} from "./server"
 export {embed_items_notebook, kernels} from "./notebook"
 export {BOKEH_ROOT, inject_css, inject_raw_css} from "./dom"
 
+type JsonItem = {doc: DocJson, root_id: string, target_id: string}
+interface Roots {[index: string]: string}
+
+export function embed_item(item: JsonItem) {
+  const docs_json: DocsJson = {}
+  const doc_id = uuid4()
+  docs_json[doc_id] = item.doc
+
+  const roots: Roots = {}
+  roots[item.root_id] = item.target_id
```

Comment:
`const roots: Roots = {[item.root_id]: item.target_id}` (aka. computed properties in ES2015).

Reply:
Thanks!